www/src/components/navigation/navbar.astro

```diff
@@ -108,3 +121,13 @@ const navbarLinks: Array<{ href: string; label: string }> = [
     window.location.pathname = "/branding";
   });
 </script>
+<style is:global>
```

Comment:
is it not possible to achieve this with tailwind classes?

Reply:
Uh tbh you're probably right, I completely forgot tailwind was on this some how.